morf-testsupport/src/main/java/org/alfasoftware/morf/jdbc/AbstractSqlDialectTest.java

```diff
@@ -5067,37 +5067,39 @@ public void testPrepareStatementParameter() throws SQLException {
 
   @Test
   public void testPortableFunction() {
-    AliasedField function = PortableFunction.withFunctionForDatabaseType(
-            "PGSQL",
-            "TRANSLATE",
-            new FieldReference("field"),
-            new FieldLiteral("1"),
-            new FieldLiteral("A"))
-        .addFunctionForDatabaseType(
-            "H2",
-            "TRANSLATE",
-            new FieldReference("field"),
-            new FieldLiteral("2"),
-            new FieldLiteral("B"))
-        .addFunctionForDatabaseType(
-            "ORACLE",
-            "TRANSLATE",
-            new FieldReference("field"),
-            new FieldLiteral("3"),
-            new FieldLiteral("C"))
-        .addFunctionForDatabaseType(
-            "MY_SQL",
-            "TRANSLATE",
-            new FieldReference("field"),
-            new FieldLiteral("4"),
-            new FieldLiteral("D"))
-        .addFunctionForDatabaseType(
-            "SQL_SERVER",
-            "TRANSLATE",
-            new FieldReference("field"),
-            new FieldLiteral("5"),
-            new FieldLiteral("E"))
-        .as("field");
+    AliasedField function = PortableSqlFunction.builder()
+            .withFunctionForDatabaseType(
+                    "PGSQL",
+                    "TRANSLATE",
+                    new FieldReference("field"),
+                    new FieldLiteral("1"),
+                    new FieldLiteral("A"))
+            .withFunctionForDatabaseType(
+                    "H2",
+                    "BTRIM",
+                    new FieldReference("field"),
+                    new FieldLiteral("2"),
+                    new FieldLiteral("B"))
+            .withFunctionForDatabaseType(
+                    "ORACLE",
+                    "REGEX_REPLACE",
+                    new FieldReference("field"),
+                    new FieldLiteral("3"),
+                    new FieldLiteral("C"))
+            .withFunctionForDatabaseType(
+                    "MY_SQL",
+                    "REVERSE",
+                    new FieldReference("field"),
+                    new FieldLiteral("4"),
+                    new FieldLiteral("D"))
+            .withFunctionForDatabaseType(
+                    "SQL_SERVER",
+                    "SOUNDEX",
+                    new FieldReference("field"),
+                    new FieldLiteral("5"),
+                    new FieldLiteral("E"))
+            .as("field")
+            .build();

```

Comment:
You should also get the alias via `PortableSqlFunction.builder().....build().as("alias")`
Testing that path is probably slightly more important, because that is what users would use out of habit.

Reply:
Following the pattern of implementing AliasedFieldBuilder